static/styles/compose.css

```diff
@@ -536,3 +557,56 @@ a.compose_control_button.hide {
         margin-left: 5px;
     }
 }
+
+#compose.compose-fullscreen {
+    height: 100%;
+    z-index: 98;
+}
+
+.compose-fullscreen {
+    #compose-container {
+        margin-top: 50px;
+        height: calc(100% - 50px);
+    }
+
+    #compose-content {
+        height: 100%;
+        display: flex;
+        flex-flow: column;
```

Comment:
What is the `flex` logic needed for?

Reply:
flex logic is needed for automatically setting the maximum height possible in the compose box. Like I give a height to #compose and then, make flex inside it. The element that needs to be stretched are given flex:1 to automtically fill all the spaces left. and height: 100% so, that height comes down. 
This makes it possible to make the height fit according to percentage of height available and parent elements.

I changed the approach as:
1. The alerts get handled automatically.
2. The elements position itself, so no need to calculate height.
3. This way also makes the height change automatically even without autosize in full screen mode if the device height is changed.

Reply:
I will see for the refracter tomorrow.

Reply:
This is possible, clearly. Will make a separate prep commit for it, mainly for handling CSS.

Reply:
I have added another commit, so that you can see if that looks good.